File: Project Euler/66.cs
using System;
using System.IO;

class dkafldjaf
{
	static void Main()
	{
		ulong max = 0;
		for ( ulong i = 1 ; i <= 1000 ; i++ )
		{
			if ( Math.Sqrt(i)%1 == 0)
			{
				continue;
			}
			for ( ulong j = 2 ; ; j++ )
			{
				if ((j*j-1)%i==0&&Math.Sqrt((j*j-1)/i)%1==0)
				{
					if ( j>max)
					{
						Console.WriteLine(i+" " + j);
						max=j;
					}
					break;
				}
			}
		}
				
	}
}

File: Project Euler/66.py
#! python
import sys
import math

d = dict()

for i in range(math.ceil(math.sqrt(1000))):
    d[i**2] = 1


for i in range(1000):
    if i in d:
        continue

#https://en.wikipedia.org/wiki/Pell%27s_equation